src/plots/cartesian/layout_attributes.js

```diff
@@ -593,6 +593,15 @@ module.exports = {
             '*%H~%M~%S.%2f* would display *09~15~23.46*'
         ].join(' ')
     },
+    hovertitle: {
+        valType: 'string',
+        dflt: '',
```

Comment:
Remove this line. The `hovertitle` default isn't blank. As it depends on which axis where coercing, we can't declare a default value here, so the best thing we can do is to not declare any default value.